SofaKernel/modules/SofaBaseMechanics/src/SofaBaseMechanics/DiagonalMass.cpp

```diff
@@ -33,6 +33,7 @@ namespace sofa::component::mass
 
 using sofa::core::objectmodel::ComponentState ;
 
+using namespace sofa::type;
 using namespace sofa::defaulttype;
```

Comment:
Is `using namespace sofa::defaulttype;` still useful?

Reply:
Actually in that particular file yes (look at the explicite instanciations with RigidTypes, etc) but for sure there are a lot of files where it wont be useful for sure (but I did not really want to check for the bazillon number of  files)